docs/modules/about/pages/introduction.adoc

```diff
@@ -0,0 +1,44 @@
+:imagesdir: ../assets/images
```

Comment:
@Bonrob2 I've just noticed, the `assets/images` directory is in `pages`. In the other section directories is it a directory higher. To make it work here it must be `assets/images`.

Reply:
Ugh. Yes. You are right. I will fix that.